src/script/page/LeftSidebar/panels/ListWrapper.tsx

```diff
@@ -56,93 +56,90 @@ interface LeftListWrapperProps {
   id: string;
   onClose?: () => void;
   hasHeader?: boolean;
+  conversationsFilter?: string;
 }
 
-const ListWrapper = ({
-  id,
-  header,
-  sidebar,
-  headerElement,
-  onClose,
-  children,
-  hasHeader = true,
-  footer,
-  before,
-  headerUieName,
-}: LeftListWrapperProps) => {
-  const calculateBorders = throttle((element: HTMLElement) => {
-    window.requestAnimationFrame(() => {
-      if (element.offsetHeight <= 0 || !isScrollable(element)) {
-        element.classList.remove('left-list-center-border-bottom', 'conversations-center-border-top');
+const ListWrapper = forwardRef<HTMLElement, LeftListWrapperProps>(
+  ({id, header, sidebar, headerElement, onClose, children, hasHeader = true, footer, before, headerUieName}, ref) => {
+    const calculateBorders = throttle((element: HTMLElement) => {
+      window.requestAnimationFrame(() => {
+        if (element.offsetHeight <= 0 || !isScrollable(element)) {
+          element.classList.remove('left-list-center-border-bottom', 'conversations-center-border-top');
+          return;
+        }
+
+        element.classList.toggle('left-list-center-border-top', !isScrolledTop(element));
+        element.classList.toggle('left-list-center-border-bottom', !isScrolledBottom(element));
+      });
+    }, 100);
+
+    function initBorderedScroll(element: HTMLElement | null) {
+      if (!element) {
         return;
       }
 
-      element.classList.toggle('left-list-center-border-top', !isScrolledTop(element));
-      element.classList.toggle('left-list-center-border-bottom', !isScrolledBottom(element));
-    });
-  }, 100);
-
-  function initBorderedScroll(element: HTMLElement | null) {
-    if (!element) {
-      return;
+      // @ts-ignore
```

Comment:
this is usually a yellow flag. why is the type not detected correctly?

Reply:
overlooked? It needs to be addressed for the tests in any case